Wikipedia/Images/WMFImageDownload.swift

```diff
@@ -12,6 +12,7 @@ public enum ImageOrigin: String {
     case Network = "Network"
     case Disk = "Disk"
     case Memory = "Memory"
+    case None = "None"
```

Comment:
Ah right, was using it at one point but then realized I didn't, but forgot to remove it. Will do.